fe2-android/app/build.gradle

```diff
@@ -186,6 +186,11 @@ copy {
     include 'examples/**'
     include 'query/**'
     include 'jsonRPC.json'
+
+    // Replace the remote part of the base URL ($id) to the local resource path
+    filter {
+        String line -> line.replaceAll("https://raw\\.githubusercontent\\.com/dedis/[a-zA-Z0-9_-]+/master/", "resource:/")
+    }
```

Comment:
out of curiosity, how come you need this?

JSON Schema defines the `$id` field as having to be a canonical URI, I believe this replacement violates the standard.

I'm very open to a pragmatic violation though if (1) there's a good enough reason, (2) it's well documented.

Reply:
The problem was that when the json validation in the fe2 is done, the schemas was correctly loaded from the local file but then the sub-schemas (from the $ref) were loaded from the online github master. (issue#630)
If a schemas was modified in a branch, it wouldn't work because it's the one from github master that would be loaded.
The library used (com.networknt:json-schema-validator) provide some ways to only use local file but it needs a mapping for each URI.
I found that it was easier to just modify the $id in the local copy, but maybe there are better solution.
The local files with theses modifications are only used to validate json.

Reply:
I was failing the CI for FE2 and I noticed that Arnaud had the same failing test 2 weeks ago so I asked him if he found a solution. He answered that I could cherry-pick the following commit: https://github.com/dedis/student_21_pop/pull/574/commits/4e86313492eb920a485ed520c467036dbc6ef418. The problem is well explained by Arnaud in this issue: https://github.com/dedis/student_21_pop/issues/630. 

In the go-backend, we have the same behaviour so I am very curious about a solution. However, I think it would be better to move this thread to PR #574.

Reply:
Oups, didn't see your comment before writing mine @arnauds5. Thanks for the explanation!

Reply:
Understood ! Thanks for the explanation.